src/planscape/stands/management/commands/load_metrics.py

```diff
@@ -0,0 +1,55 @@
+from django.core.management.base import BaseCommand, CommandParser
+from conditions.models import Condition
+from django.db import connection
+
+
+class Command(BaseCommand):
+    help = "Loads stand metrics based on conditions existing in the database."
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "--condition-ids",
+            nargs="+",
+            type=int,
+        )
+
+        parser.add_argument(
+            "--clear",
+            default=True,
+            action="store_true",
+            help="If set to false, the command will not remove previous metrics data from each condition from the raster table.",
+        )
+
+    def handle(self, *args, **options):
+        condition_ids = options.get("condition_ids")
+        clear = options.get("clear")
+        if condition_ids:
+            conditions = Condition.objects.filter(id__in=condition_ids)
+        else:
+            conditions = Condition.objects.all()
+
+        results = list(
+            [self.handle_condition(condition, clear) for condition in conditions]
+        )
+        total = len(results)
+        success = len([result for result in results if result[0]])
+        failure = total - success
+
+        self.stdout.write(
+            f"Stand Metrics Loaded: {success}.\nStand Metrics Failed: {failure}"
```

Comment:
Do we want to log somewhere which conditions succeeded and which conditions failed? Or is the idea that if there are any failures you would want to rerun the whole thing

Reply:
looks like the intention is to output something on line 53.